<launch>
    <node name="ground_station" output="screen" pkg="magna" type="GroundStation.py" />
</launch>